run.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+set -e
+
+if [ -d "$DIRECTORY" ]; then
+    echo 'Cloning roles'
+    git clone git@github.com:personal-provisioning/roles.git ../roles
```

Comment:
Why clone it into `../roles`. The last approach was to clone it into `~/.personal_provisioning/roles`, so we can use this patch in our ansible config.

Reply:
A standardized way makes sense, you are right 👍 will change it